gamepad_test.go

```diff
@@ -0,0 +1,104 @@
+package uinput_test
+
+import (
+	"testing"
+	"time"
+	"uinput"
```

Comment:
My tests to this file don't pass, and the only way I can find to make this work is to change this import to
`"github.com/bendahl/uinput"`

Reply:
Which version of golang are you using?

Reply:
go version go1.18.3 linux/amd64, I also add that I am working with go modules